src/golang/cmd/server/handler/create_table.go

```diff
@@ -185,5 +184,9 @@ func CreateTable(ctx context.Context, args *CreateTableArgs, contentPath string,
 		return http.StatusInternalServerError, errors.Wrap(err, "Unable to create table.")
 	}
 
-	return http.StatusBadRequest, errors.Newf("Unable to create table: %v", metadata.Error)
+	errMsg := ""
+	if metadata.Error != nil {
```

Comment:
nit: shouldn't we raise an internal exception if `metadata.Error == nil` here? It's expected that we have an error at this point right.